src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+class Square extends Figure {
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    private double side;
+
+    @Override
+    public double area() {
```

Comment:
The method name 'area' should be a verb, for example, 'getArea', to be more descriptive and to follow Java method naming conventions.